tests/functional/conversion/test_encode_structs.py

```diff
@@ -0,0 +1,44 @@
+from ethpm_types.abi import MethodABI
+from hexbytes import HexBytes
+from pydantic import BaseModel
+
+from ape import networks
+
+ABI = MethodABI.parse_obj(
+    {
+        "type": "function",
+        "name": "test",
+        "inputs": [
+            {
+                "name": "s",
+                "type": "tuple",
+                "components": [
+                    {"name": "a", "type": "uint256"},
+                    {"name": "b", "type": "bytes32"},
+                    {"name": "c", "type": "bool"},
+                ],
+            }
+        ],
+    }
+)
+
+
+class Struct(BaseModel):
```

Comment:
Ethereum does not use `https://github.com/ApeWorX/ethpm-types/blob/main/ethpm_types/abi.py#L394` does it?

Reply:
No, not yet